### ISSUE TITLE
proxy-bypass-list=<-loopback> required for Chrome for localhost use

### ISSUE THREAD

**Issue by rra** [wkeeling/selenium-wire#157]
If Selenium is used to make requests to a server running on localhost (for end-to-end testing of a web application, for example), selenium-wire appears to work but is silently bypassed, so none of the functionality that it adds takes effect. This is because Chrome (at least as of version 83) automatically bypasses proxy settings for requests to localhost, so the requests bypass the selenium-wire proxy.

The magic incantation to fix this is:

```
options = webdriver.ChromeOptions()
options.add_argument("proxy-bypass-list=<-loopback>")
driver = webdriver.Chrome(options=options)
```

Could you add this to the documentation? Or, even better, perhaps it would make sense to add this option automatically when spawning Chrome unless proxy-specific options are already set?

**Comment by wkeeling** [comment]
Good find, many thanks. This should be added automatically by Selenium Wire to the Chrome specific config, as you mention. I'll look at doing that and do some testing around it.

Thanks again!

**Comment by wkeeling** [comment]
Fixed in v2.1.2
